client/src/components/Sub_ProductDetail/Sub_ImageGallery/ZoomImage.jsx

```diff
@@ -29,19 +33,10 @@ function ZoomImage({ src, modalView, setModalView }) {
   }, [modalView]);
 
   return (
-    <img
-      ref={imageRef}
-      src={src}
-      alt="main product in zoom view"
-      style={{
-        width: '250%',
-        height: '250%',
-        transform: `translate(${imageX}px, ${imageY}px)`,
-        transition: 'all 0.5s ease-out',
-        cursor: 'zoom-out',
-      }}
-      onClick={handleClick}
-    />
+    <figure onClick={handleClick} onMouseMove={handleMouseMove} style={{ backgroundImage: `url(${src})`, backgroundPosition: `${imageX}% ${imageY}%`, backgroundSize: '250%', cursor: 'zoom-out', }}>
+      <img src={src} />
+    </figure>
+
```

Comment:
This looks like code I'd like to investigate further for my own uses in "My Outfit"...